spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -37,6 +37,19 @@ However, any application registered with a Maven, HTTP, or File resource for the
 The Spring Cloud Data Flow implementation for Kubernetes uses the link:https://github.com/spring-cloud/spring-cloud-deployer-kubernetes[Spring Cloud Deployer Kubernetes] library for orchestration.
 Before you begin setting up a Kubernetes cluster, refer to the link:https://github.com/spring-cloud/spring-cloud-deployer-kubernetes#kubernetes-compatibility[compatibility matrix] to learn more about deployer and server compatibility against Kubernetes release versions.
 
+The following outlines the compatibility between Spring Cloud Data Flow for Kubernetes Server and Kubernetes versions:
+
+[source,console]
+----
+SCDF K8S Server Version \ Kubernetes | 1.8.x | 1.9.x | 1.10.x |
```

Comment:
It'd be good to also include the K8s-deployer version alongside K8s-server versions. Perhaps: `Server: 1.4.x - Deployer: 1.3.x` etc.

Reply:
Maybe also change the heading to `Versions: SCDF K8S Server - K8S Deployer \ Kubernetes`

Reply:
@sabbyanandan the `Deployer` value for all would be `1.3.x` rather than a specific version as there may not be a one to one mapping, i.e.: a newer deployer version may have been cut for a a patch release. SCDF back to 1.4.x to 1.7.x uses the 1.3.x line of deployer versions

Reply:
per conversation - for each SCDF version use the deployer version included in the latest SCDF patch release